specifications/passive/dreem-1.0.0.yml

```diff
@@ -0,0 +1,21 @@
+#====================================== For DREEM device =====================================#
+vendor: DREEM
+model: NA
+version: 1.0.0
+doc: DREEM data collected and uploaded by DREEM web application
+data:
+  - type: DREEM SLEEP STAGE
```

Comment:
I think the type property here is not relevant, thus can be removed. Other wise can you follow the syntax from other specs and append the works with an `_` and remove the DREEN prefix?